CircleSpawn/CircleSpawn/Models/Circle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Circle.swift
+//  CircleSpawn
+//
+//  Created by Taras Kulyavets on 02.04.2020.
+//  Copyright © 2020 DaftAcademy. All rights reserved.
+//
+
+import UIKit
+
+class Circle: UIView {
+
+    /*
+    // Only override draw() if you perform custom drawing.
+    // An empty implementation adversely affects performance during animation.
+    override func draw(_ rect: CGRect) {
+        // Drawing code
+    }
+    */
+    
+    var touch: CGPoint = CGPoint()
```

Comment:
1. To nie jest uniwersalna cecha tego widoku, że będziemy go tapować. Trzymanie tej zmiennej wewnątrz `ViewControllera` jest moim zdaniem lepsze.
2. Dlaczego reprezentujesz brak wartości (view nie był dotknięty jeszcze) poprzez domyślny `CGPoint` (`x: 0, y:0`), zamiast użyć optionali które są do tego stworzone i zrobić `var touch: CGPoint? = nil`?
3. Nazwa zmiennej jest bardzo mało opisowa. To nie jest touch, tylko point, i nie wiadomo co to oznacza. Nazywajcie zmienne tak, żeby nie trzeba było przeczytać całego pozostałego kodu w projekcie żeby zrozumieć, co one oznaczają.